content/TruenasApps/StableApps/InstallNextCloudMedia.md

```diff
@@ -15,151 +15,276 @@ keywords:
 - software storage solutions
 ---
 
-Nextcloud is a drop-in replacement for many popular cloud services, including file sharing, calendar, groupware and more.
+Nextcloud is a drop-in replacement for many popular cloud services, including file sharing, calendar, groupware, and more.
 One of its more common uses for the home environment is serving as a media backup, and organizing and sharing service.
-This procedure demonstrates how to set up Nextcloud on TrueNAS, and configure it to support hosting a wider variety of media file previews, including High Efficiency Image Container (HEIC), MP4 and MOV files.
-The instructions in this article apply to TrueNAS 22.12.0 and later.
+This procedure demonstrates how to set up Nextcloud on TrueNAS, and configure it to support hosting a wider variety of media file previews, including High Efficiency Image Container (HEIC), MP4, and MOV files.
+The instructions in this article apply to TrueNAS 22.10.0 and later.
 
 {{< include file="/static/includes/AppsUnversioned.md" >}}
 
 ## Before You Begin
-Before using TrueNAS to install the Nextcloud application you need to [create four datasets](#adding-nextcloud-storage) to use as storage for the Nextcloud application.
+Before you install the Nextcloud app:
 
-If you are creating a new user account to manage this application or using an [administrator]({{< relref "ManageLocalUsersSCALE.md" >}}) account, enable sudo permissions for that account.
+{{< include file="/static/includes/apps/AppsStableBeforeYouBegin.md" >}}
 
-If creating a new user for Nextcloud, add the user to the dataset ACL permissions.
+{{< include file="/static/includes/apps/AppsBeforeYouBeginCertificate.md" >}}
 
-If you want to use a certificate for this application, create a new self-signed CA and certificate, or import the CA and create the certificate if using one already configured for Nextcloud. A certificate is not required to deploy the application.
+<p style="margin-left: 33px">Adding a certificate is optional but if you want to use a certificate for this application, either create a new self-signed CA and certificate or import an existing CA and create the certificate for Nextcloud. A certificate is not required to deploy the application.</p>
```

Comment:
I structured this snippet this way because certificates are required for some apps and optional for others. If I insert the (optional) in the bullet point I cannot use the same snippet in all app tutorials. I've modified the sentence to include (if required) at the end it.